analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.cs

```diff
@@ -172,5 +189,89 @@ public void Execute_PersistSymbols_InsideBlock()
             static ISymbol LocalReferenceOperationSymbol(IOperationWrapperSonar operation) =>
                 ((ILocalReferenceOperation)operation.Instance).Local;
         }
+
+        [TestMethod]
+        public void Execute_SimpleAssignmentWithLiteralOnLocalVariables_PropagatesConstraints()
```

Comment:
We don't care about the main `Execute_` part but rather this operation as the key stuff.
Similar for others.
I've removed the `PropagateConstraints` as everyone will do that, unless said otherwise.
```suggestion
        public void SimpleAssignment_ToLocalVariable_FromLiteral()
```